share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/site-finder/site-finder.get_pt_BR.properties

```diff
@@ -15,7 +15,7 @@ site-finder.cancel-request-failure=Falha ao cancelar a solicita\u00e7\u00e3o par
 site-finder.join-success=O usu\u00e1rio {0} foi adicionado ao site {1} com sucesso
 site-finder.join-failure=Falha ao adicionar o usu\u00e1rio {0} ao site {1}
 site-finder.leave-success=O usu\u00e1rio {0} foi removido do site {1} com sucesso
-site-finder.leave-failure=Falha ao remover o usu\u00e1rio {0} do site {1}. Deve haver pelo menos um gerenciador de site por site.
+site-finder.leave-failure=Falha ao remover o usu\u00e1rio ''{0}'' do site ''{1}''. Deve haver pelo menos um Gerenciador de site por site.
```

Comment:
The right work here is Gerente, and not Gerenciador.
Gerente is a role... Gerenciador is a tool